kubectl-cloudflow/build-and-test.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 go mod tidy
-go test github.com/lightbend/cloudflow/kubectl-cloudflow/...
+go test github.com/lightbend/cloudflow/kubectl-cloudflow/... -v -tags=integration
```

Comment:
This runs integration tests as well. Images parsed with the integration tests have been pushed to dockerhub. Ideally we want the images to be build locally on the fly.